apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -40,7 +42,34 @@ const ReviewWork = ({ issue, submissionIndex, readOnly }) => {
 
   const [ feedback, setFeedback ] = useState('')
   const [ rating, setRating ] = useState(-1)
-  const [ index, setIndex ] = useState(submissionIndex)
+
+  const submissionsApproved = issue.workSubmissions.reduce((counter, submission) => {
+    if ('review' in submission && submission.review.accepted) counter++
```

Comment:
I find this mix of `'thing' in otherThing` and `otherThing.thing.whatever` within the same conditional particularly odd.

Also, I think it might be easier to read & work with if we get rid of the `reduce`:

```js
const approvedSubmissions = issue.workSubmissions.filter(s => s.review && s.review.accepted)

// then later:
approvedSubmissions.length

Reply:
That's much better, thank you.